### ISSUE TITLE
keepass-diff main thread panicks

### ISSUE THREAD

**Issue by frunzales** [Narigo/keepass-diff#8]
See below

```
xxx@x230 ~/kpx $ RUST_BACKTRACE=1 ~/.cargo/bin/keepass-diff a/gah2Hohp.kdbx b/lachooJ1.kdbx --passwords pass



thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:345:21
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39
   1: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:71
             at src/libstd/sys_common/backtrace.rs:59
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:197
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:211
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:474
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:381
   6: rust_begin_unwind
             at src/libstd/panicking.rs:308
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
   8: core::panicking::panic
             at src/libcore/panicking.rs:49
   9: keepass_diff::diff::compare
  10: keepass_diff::main
  11: std::rt::lang_start::{{closure}}
  12: std::panicking::try::do_call
             at src/libstd/rt.rs:49
             at src/libstd/panicking.rs:293
  13: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:87
  14: std::rt::lang_start_internal
             at src/libstd/panicking.rs:272
             at src/libstd/panic.rs:388
             at src/libstd/rt.rs:48
  15: main
  16: __libc_start_main
  17: _start
xxx@x230 ~/kpx $
```

**Comment by Narigo** [comment]
I have merged the open pull request where I also fixed a bug regarding different length databases. Could you retry with the latest version from master, please?

**Comment by Narigo** [comment]
Sorry, I think this was due to me forgetting to publish correctly... 😖 

Can you retry, please? I just released version `0.2.1` as well. If it still doesn't work, can you make a reproducer? I think in earlier versions this could happened when the files did not have the same amount of entries in them. But it should be fixed with version `0.2.0` and up.

**Comment by frunzales** [comment]
After doing a quick test I can confirm that I don't get a crash anymore. This is safe for closing.
